core/database/arango/arango_unix_client.py

```
@@ -0,0 +1,157 @@
+    """
+    ArangoDB client using Unix socket connection.
+
+    Uses the Unix socket at /tmp/arangodb.sock for local connections,
+    providing better performance than HTTP connections.
+
+    Theory Connection (Conveyance Framework):
+    Optimizes the TIME (T) dimension by minimizing network latency,
+    directly improving Conveyance C = (W·R·H/T)·Ctx^α
+    """
+
+    SOCKET_PATH = "/tmp/arangodb.sock"
```

Ruff (0.12.2)

Probable insecure usage of temporary file or directory: "/tmp/arangodb.sock"

(S108)